torchvision/extension.py

```diff
@@ -16,6 +16,18 @@ def _has_ops():
 
 
 try:
+    # On Windows Python-3.8+ has `os.add_dll_directory` call,
+    # which is called to configure dll search path.
+    # To find cuda related dlls we need to make sure the
+    # conda environment/bin path is configured Please take a look:
+    # https://stackoverflow.com/questions/59330863/cant-import-dll-module-in-python
+    if os.name == "nt" and sys.version_info >= (3, 8) and sys.version_info < (3, 9):
+        env_path = os.environ["PATH"]
+        path_arr = env_path.split(";")
+        for path in path_arr:
+            if os.path.exists(path):
+                os.add_dll_directory(path)
```

Comment:
Looks like the [linter](https://app.circleci.com/pipelines/github/pytorch/vision/20890/workflows/a16f22e3-b412-482b-b4d1-d3fff416db84/jobs/1698482) is failing because it runs on Linux. We might need something like:
```suggestion
                os.add_dll_directory(path)  # type: ignore[attr-defined]
```